lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -1060,14 +1060,15 @@ def get(self, request, subsection_id):
             original_grade = PersistentSubsectionGrade.read_grade(user_id, usage_key)
             if original_grade is not None and hasattr(original_grade, 'override'):
                 override = original_grade.override
-                history = list(original_grade.override.history.all().order_by('history_date')[:history_record_limit])
+                history = PersistentSubsectionGradeOverride.history.filter(grade_id=original_grade.id).order_by(
```

Comment:
I think you still have to wrap this in `list()` to evaluate the queryset: https://docs.djangoproject.com/en/1.11/ref/models/querysets/#when-querysets-are-evaluated